host-configuration/docs/3_AGL_XDS.md

```diff
@@ -0,0 +1,24 @@
+# AGL XDS
```

Comment:
Make no sens to install both agent and server on the same host/machine.
For this section (AGL XDS), I would just add some explanations and add a link to XDS User's Guide doc:
 http://docs.automotivelinux.org/docs/devguides/en/dev/reference/xds/part-1/1_install-client.html
 http://docs.automotivelinux.org/docs/devguides/en/dev/reference/xds/part-1/2_install-xds-server.html

Reply:
the first idea with Ronan was to do the opposite, add a link into these two parts so as to not depend on repo in those sections.
maybe we can separate client/server side in installation?